### DISCUSSION TITLE
Crosspoint

### DISCUSSION THREAD

**Discussion by thesnable** [vit1251/golden#184] [General]
Hi!

So how does this compare for example to crosspoint?

Can i use this instead of?

Thanks alot
marco

**Comment by vit1251** [comment]
> So how does this compare for example to crosspoint?
> Can i use this instead of?

I'm not familiar with crosspoint. In any case, if you have a point account with your provider that distributes mail using the binkp protocol, then you can use Golden Point with netmail and echomail messaging.

The documentation also indicates the standards that aplication would like to respects, so you can clarify or modify some of these to suit your needs.
